src/OpalCompiler-Core/CompilationContext.class.st

```diff
@@ -542,11 +542,9 @@ CompilationContext >> noPattern [
 
 { #category : #accessing }
 CompilationContext >> noPattern: anObject [
-	anObject ifTrue: [
-		semanticScope := semanticScope asDoItScope.
-		"when we compile an expression, embedd sources, as the resulting method will never be installed"
-		self parseOptions: #(+ #optionEmbeddSources).
```

Comment:
Is this parse option removal intended?

Reply:
Yep, it is wrongly placed and duplicates one that is correctly placed: https://github.com/pharo-project/pharo/blob/Pharo12/src/OpalCompiler-Core/OpalCompiler.class.st#L335